parsec.tf

```diff
@@ -24,102 +16,143 @@ provider "aws" {
 
 data "aws_ami" "parsec" {
   most_recent = true
+
   filter {
-    name = "name"
-    values = ["parsec-ws2016-6"]
+    name   = "name"
+    values = ["parsec-g3-ws2016-10"]
   }
 }
 
+resource "aws_vpc" "vpc_parsec" {
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name = "Parsec VPC"
+  }
+}
+
+resource "aws_subnet" "vpc_parsec_public_subnet" {
+  availability_zone       = "us-east-1a"
```

Comment:
I think this should be set to use region set in variable `${var.aws_region}`

Reply:
Good catch! Missed that. I am now passing in the the cheapest AZ.